os/binfmt/elf.c

```diff
@@ -306,7 +306,7 @@ static int elf_loadbinary(FAR struct binary_s *binp)
 	binp->alloc[4] = (FAR void *)loadinfo.dataalloc;
 	binp->textsize = loadinfo.textsize;
 	binp->rosize = loadinfo.rosize;
-	binp->datasize = loadinfo.datasize;
+	binp->datasize = loadinfo.datasize - binp->bsssize;
```

Comment:
Could you remain comments why we subtract bsssize?

Reply:
Done.